### DISCUSSION TITLE
Ability to disable extension per project

### DISCUSSION THREAD

**Discussion by thinkstudeo** [vuedx/languagetools#193] [Q&A] [answered]
I want to try this extension for newer Vue 3 based projects which are written in typescript.

However I also have older Vue 2 based projects which are not written in typescript. And using Vetur for the older projects is fine. I can disable Vetur for the newer projects as it provides option to enable/disable Vetur for a particular project.

It would be really helpful and nice if the ability to disable VueDx for older Vue 2 based projects not written in typescript is made available.

It would allow to keep using Vetur for the older projects till I have to support them or upgrade to Vue 3 and typescript and for all new Vue 3 based projects I can use VueDx.

@znck Thank you very much for your work on this extension, it really looks very very promising and am looking forward to being able to use it.

**Comment by mika76** [comment, marked as answer]
You should already be able to disable extensions per workspace in vscode - https://code.visualstudio.com/docs/editor/extension-gallery#_disable-an-extension
